ibm_mq/assets/configuration/spec.yaml

```diff
@@ -191,6 +191,16 @@ files:
         value:
           example: False
           type: boolean
+      - name: try_normal_connection
```

Comment:
Can we clarify in the description that this connection tries with `username` and `password` before SSL when `ssl_auth` is enabled? 

Also, just a nit, `normal connection` sounds a little ambiguous, but I see that it's based on the method name. Perhaps we can change the option name to `try_basic_connection` or `try_basic_auth`?

Reply:
True, `try_basic_auth` sounds better!